components/MedicationCapturePage.js

```diff
@@ -316,12 +329,26 @@ class MedicationCapturePage extends Component {
                 });
             }
         }
+
+        audioNegative(){
+            if(song != null){
```

Comment:
Can we change the name `song` to something a little more descriptive? And the variable is also never declared, can we do this as well?